### PR TITLE
Add BlueSolo

### PR DESCRIPTION
**Pull request by NatsuDzn**

### CODE REVIEW

File: src/pages/BlueSolo/main.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { pageInterface } from '../pageInterface';
+
+export const BlueSolo: pageInterface = {
+  name: 'BlueSolo',
+  domain: 'https://www.bluesolo.org/',
```
Comment (henrik9999) [inline src/pages/BlueSolo/main.ts:5]: remoe the / at the end

**Review by henrik9999 — CHANGES_REQUESTED** [review]

File: src/pages/BlueSolo/main.ts (hunk 1 above)
Comment (NatsuDzn) [reply src/pages/BlueSolo/main.ts]: Done ✅

File: src/pages/BlueSolo/meta.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "urls": {
+    "match": ["*://www.bluesolo.org/*"]
```
Comment (lolamtisch) [inline src/pages/BlueSolo/meta.json:3]: Why not directly use `*://www.bluesolo.org/manga/*`

File: src/pages/BlueSolo/main.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      }
+      if (type.includes('one-shot')) {
+        return 1;
+      }
+      return NaN;
+    },
+    nextEpUrl() {
+      const nextChapter = j
+        .$(
+          '.header > .wp-manga-nav > .select-view > .chapters_selectbox_holder > .selectpicker_chapter > label > select > option:selected',
```
Comment (lolamtisch) [inline src/pages/BlueSolo/main.ts:37]: This selector is too specific.
I think `.selectpicker_chapter select option:selected` should be enough

**Review by lolamtisch — DISMISSED** [review]

File: src/pages/BlueSolo/meta.json (hunk 2 above)
Comment (NatsuDzn) [reply src/pages/BlueSolo/meta.json]: Done ✅

File: src/pages/BlueSolo/main.ts (hunk 3 above)
Comment (NatsuDzn) [reply src/pages/BlueSolo/main.ts]: Yeah it's enough

**Review by henrik9999 — APPROVED** [review]

**Pull request merged**